### PR TITLE
Bug fix: `mondo#ABBREVIATION` -> `OMO:0003000`

### PR DESCRIPTION
**Pull request by joeflack4**
partially addresses https://github.com/monarch-initiative/mondo-ingest/issues/655

## Changes
Bug fix: mondo#ABBREVIATION
- Change to `OMO:0003012`

### CODE REVIEW

**Comment by twhetzel** [comment]
@joeflack4 did you confirm that this fixes running a full build on `develop` or `main` in mondo-ingest?

File: omim2obo/namespaces.py [hunk 1]
```diff
@@ -56,6 +56,7 @@
 OBOINOWL = Namespace('http://www.geneontology.org/formats/oboInOwl#')         # obo-specific annotation properties, like synonym types
```
Comment (twhetzel) [inline omim2obo/namespaces.py:56]: FYI - I think there is an earlier PR that changes this back to `oboInOwl`

File: omim2obo/main.py [hunk 2]
```diff
@@ -220,7 +220,7 @@ def omim2obo(use_cache: bool = False):
             graph.add((axiom, OWL.annotatedSource, omim_uri))
             graph.add((axiom, OWL.annotatedProperty, oboInOwl.hasExactSynonym))
             graph.add((axiom, OWL.annotatedTarget, Literal(abbreviation)))
-            graph.add((axiom, OBOINOWL.hasSynonymType, MONDONS.abbreviation))
+            graph.add((axiom, OBOINOWL.hasSynonymType, OMO['0003000']))
```
Comment (twhetzel) [inline omim2obo/main.py:223]: `OBOINOWL` --> `oboInOwl`

Thanks for changing to the property `OMO_0003000`.

**Review by twhetzel — APPROVED** [review]
The changes look good and I see the new abbreviation property in the "omim.owl" file in https://github.com/monarch-initiative/omim/releases/tag/2024-09-20. 

Did this fix the full build in mondo-ingest?

File: omim2obo/main.py (hunk 2 above)
Comment (joeflack4) [reply omim2obo/main.py:223]: Hmm I didn't think I'd made that change here yet. I'll change it back.

**Comment by joeflack4** [comment]
> @joeflack4 Did this fix the full build in mondo-ingest?

Yep!

File: omim2obo/main.py (hunk 2 above)
Comment (joeflack4) [reply omim2obo/main.py:223]: Oh! I didn't actually change that in here.

I remember now; there were duplicate namespaces; one `oboInOwl` and another `OBOINOWL`. Anywhere there's only the former now.

**Pull request merged**